BadHabbit/BadHabbit/UserStories/SingUpScreen/SignUpScreenRouter.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SignUpScreenRouter.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 24/07/2023.
+//
+
+import Foundation
+
+protocol SignUpRouterProtocol {
+    func presentAnotherScreen()
+}
+
+final class SignUpScreenRouter: SignUpRouterProtocol {
+    
+    let nextScreen: StoryAssembly
+    let globalCoordinator: IGlobalCoordinator
+    
+    init(nextScreen: StoryAssembly,
+         globalCoordinator: IGlobalCoordinator) {
+        self.nextScreen = nextScreen
+        self.globalCoordinator = globalCoordinator
+     }
+
+    func presentAnotherScreen() {
+        let vc = nextScreen.assembleStory()
+        globalCoordinator.presentOnTopVisibleViewController(vc, animated: true, completion: nil)
```

Comment:
you can use convenient function without params animated and completion

Reply:
Did you mean that I can add my own function into GlobalCoordinator class or I can set the default value of the animated parameter as true or something else (in the context of our app architecture)?

Reply:
I propose to add new function with only one parameter - ViewController

Reply:
Function was added